charts/groups/templates/applications.yaml

```
@@ -14,10 +17,12 @@ spec:
   project: {{ $group.name }}
   source:
     repoURL: {{ .repoURL }}
-    path: {{ .path }}
-    targetRevision: {{ .targetRevision }}
+    path: {{ $path }}
+    targetRevision: {{ $target }}
```

Not sure why I suggested `$path` / `$target` when this could be inlined

```suggestion
    path: {{ default "." .path }}
    targetRevision: {{ default "HEAD" .targetRevision }}
```